docs/docsource/Operation-Architecture/Orbit_UI/Bucket_Management/Lifecycle_Management.rst

```diff
@@ -1,45 +1,39 @@
 Object Lifecycle Management
 ===========================
 
-**Prerequisite:** You must have established at least one bucket.
-
-#. From anywhere in Orbit, click the **WORKFLOWS > Bucket
-   Lifecycle** tab in the left navbar.
-
-   |image0|
-
-#. The **Bucket Lifecycle** screen displays.
-
-   |image1|
-
-#. Choose a bucket and pick **Add New Rule > Expiration**
-#. The **Add New Rule** dialog appears:
-
-   |image2|
-
-   You may enter a distinct directory or subdirectory to which the rule applies.
-   Enter an expiration time span and a deletion time span.
-   These follow the bucket and enforce expiration and deletion.
-   You may also add a comment about this expiration rule.
-
-   Click **Save**.
-
-#. The new rule is displayed:
-
-   |image3|
-
-   Zenko will enforce these rules on this bucket. If replication is
-   configured, any change of state to objects in this bucket can be
-   replicated to buckets on other clouds.
-
-Versioning logic precludes simply deleting an object: that day’s object
-is deleted, but all others remain. See warning at “Deleting Files,” on
-page 1.
-
-.. |image0| image:: ../../Resources/Images/Orbit_Screencaps/Orbit_lifecycle_select.png
-.. |image1| image:: ../../Resources/Images/Orbit_Screencaps/Orbit_lifecycle_bucket_select.png
-   :class: OneHundredPercent
-.. |image2| image:: ../../Resources/Images/Orbit_Screencaps/Orbit_lifecycle_add_rule.png
-   :class: FiftyPercent
-.. |image3| image:: ../../Resources/Images/Orbit_Screencaps/Orbit_lifecycle_rule_success.png
-   :class: OneHundredPercent
+Orbit enables you to set policies that manage lifecycle events in selected
+buckets. 
+
+From the Bucket Lifecycle selector on the Zenko sidebar menu, you can select,
+then create lifecycle transition or lifecycle expiration policies for objects
+in buckets. These transition (move) or expire (delete) objects that match your
+criteria based on a timespan you set. In other words, Zenko reviews when a set
+number of days has passed since an object or type of object was last touched,
+and either moves such objects to a different storage site (transitions it) or 
+deletes (expires) them. You can create rules to transition objects themselves, 
+or if versioning is enabled, to transition object versions.
+
+This facility offers responsiveness to needs for data freshness and economy. 
+You can, for example, store job information for ongoing customer interactions
+on a local RING. After a customer stops making demands for immediate (low-
+latency) support, information pertaining to their purchase order transitions
+from the company RING to a public cloud. Latency may increase, but cloud data
+storage costs may be reduced. After a few months, if the data on the public
+cloud is not accessed, it can either be transitioned to another cloud or 
+deleted (expired). 
+
+Orbit supports most of the Amazon S3 lifecycle management command set; 
```

Comment:
It sounds like some of these are on the roadmap to be added. So I'm not sure whether we want to include it in a static release doc because Orbit could roll out these at a later date. @RachedBenMustapha @nicolas2bert Can you all weigh in on that?

Reply:
I don't see any way around this. We can (and in fact do) describe what we *do* at great length, but the thing that interests the user here is the short list of things they *can't* do. In terms of maintenance, if there are tickets in place for these proposed features, perhaps we could drop in a task to remove the pertinent "can't do it" from the documentation for each feature?

Reply:
I agree. Just thinking that if it changes, we'd maybe need a way to update this. It's an open question for me. 🤔

Reply:
I'm going to score this as "resolved (for now...)"